<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1366</width>
      <height>768</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page1">
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>130</y>
        <width>791</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="WeekSched_2">
       <item>
        <widget class="QFrame" name="WeekFrame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFrame" name="WeekFrame2_2">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Weekly Schedule</string>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignRight">
              <widget class="QLineEdit" name="StudentSearch">
               <property name="placeholderText">
                <string>Enter Student ID</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Search">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Search</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="viewCurriculum">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>View Curriculum</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="Semester">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <item>
                <property name="text">
                 <string>1st Semester</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2nd Semester</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="WeekTable_2">
            <row>
             <property name="text">
              <string>7:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>8:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>9:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>10:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>11:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>12:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>1:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>2:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>3:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>4:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>5:00AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>5:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>7:00 PM</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Sunday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Monday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Tuesday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Wednesday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Thursday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Friday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Saturday</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>276</width>
        <height>491</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="TodaySched_3">
       <item>
        <widget class="QFrame" name="TodayFrame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item alignment="Qt::AlignLeft|Qt::AlignTop">
           <widget class="QLabel" name="TodaySched_4">
            <property name="text">
             <string>Today's Schedule</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter|Qt::AlignTop">
           <widget class="QLabel" name="labelTodayHeader_2">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QTableWidget" name="tableWidget_2">
            <row>
             <property name="text">
              <string>7:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>8:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>9:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>10:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>11:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>12:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>1:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>2:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>3:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>4:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>5:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>6:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>7:00 PM</string>
             </property>
            </row>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="ScheduleTitle">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>50</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Schedule</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page2"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
